.github/workflows/linux.yml

```diff
@@ -16,11 +16,13 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job
       # can access it
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - name: setup prerequisites
         run: |
           sudo apt update
-          sudo apt install autoconf automake libtool libudev-dev m4
+          sudo apt install autoconf automake libtool libudev-dev m4 clang-tidy clang wget
 
```

Comment:
If clang is added here just because it is a dependency of clang-tidy, it is not needed here. apt takes care of dependencies.